client/ayon_core/tools/publisher/widgets/list_view_widgets.py

```diff
@@ -454,6 +457,7 @@ def __init__(self, controller, parent):
         instance_view.collapsed.connect(self._on_collapse)
         instance_view.expanded.connect(self._on_expand)
         instance_view.toggle_requested.connect(self._on_toggle_request)
+        instance_view.doubleClicked.connect(self._on_double_clicked)
```

Comment:
Is there a particular reason this isn't just like this and don't need the `_on_double_clicked` in-between?
```suggestion
        instance_view.doubleClicked.connect(self.double_clicked)
```

Reply:
no there is no particular reason why.
I implemented the suggestion.